frontend/projects/admin/src/app/event/event-dashboard/event-dashboard.component.html

```diff
@@ -380,32 +391,424 @@ <h3 class="mt-5">
+                    >
+                      {{ ticketCategory.checkedInTickets }}
+                    </div>
+                    <div
+                      *ngIf="ticketCategory.soldTickets != 0"
+                      class="progress-bar bg-success"
+                      role="progressbar"
+                      [style]="
+                        'width: ' +
+                        (ticketCategory.soldTickets /
+                          getActualCapacity(
+                            ticketCategory,
+                            eventOrganizationInfo.event
+                          )) *
+                          100 +
+                        '%'
+                      "
+                    >
+                      {{ ticketCategory.soldTickets }}
+                    </div>
+                    <div
+                      *ngIf="ticketCategory.pendingTickets != 0"
+                      class="progress-bar bg-info"
+                      role="progressbar"
+                      [style]="
+                        'width: ' +
+                        (ticketCategory.pendingTickets /
+                          getActualCapacity(
+                            ticketCategory,
+                            eventOrganizationInfo.event
+                          )) *
+                          100 +
+                        '%'
+                      "
+                    >
+                      {{ ticketCategory.pendingTickets }}
+                    </div>
+                    <div
+                      *ngIf="ticketCategory.notSoldTickets != 0"
+                      class="progress-bar bg-info"
+                      role="progressbar"
+                      [style]="
+                        'width: ' +
+                        (ticketCategory.notSoldTickets /
+                          getActualCapacity(
+                            ticketCategory,
+                            eventOrganizationInfo.event
+                          )) *
+                          100 +
+                        '%'
+                      "
+                    >
+                      {{ ticketCategory.notSoldTickets }}
+                    </div>
+                    <div
+                      *ngIf="!ticketCategory.bounded"
+                      type="default"
+                      class="progress-bar bg-info"
+                      role="progressbar"
+                      [style]="
+                        'width: ' +
+                        (eventOrganizationInfo.event.dynamicAllocation /
+                          getActualCapacity(
+                            ticketCategory,
+                            eventOrganizationInfo.event
+                          )) *
+                          100 +
+                        '%'
+                      "
+                    >
+                      {{ eventOrganizationInfo.event.dynamicAllocation }}
+                    </div>
+                  </div>
+                </div>
+
```

Comment:
`checkedInTickets` bar color should be **success** (`bg-success`).
`soldTickets` should be **warning**
`pendingTickets`/`notSoldTickets` should be **info** **(already OK)**
`dynamicAllocation` should be **default** (i.e. no "bg-" class)